### PR TITLE
Updated to the new UI

### LINKED ISSUE #1497: Managing Resource Servers chapter
### Description

Update this Authorization Services Guide chapter for the new Admin Console

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by edewit**
fixes: #1497

### CODE REVIEW

File: authorization_services/topics/resource-server-create-client.adoc [hunk 1]
```diff
@@ -10,12 +10,22 @@ The first step to enable {project_name} Authorization Services is to create the
...
+endif::[]
 +
 .Add Client
 image:{project_images}/resource-server/client-create.png[alt="Add Client"]
 
 . Type the `Client ID` of the client. For example, _my-resource-server_.
+ifeval::[{project_community}==true]
+. Click *Next*.
+. Toggle *Client authentication* to ON.
+. Click *Save*.
```
Comment (andymunro) [inline authorization_services/topics/resource-server-create-client.adoc:27]:
```suggestion
. Click *Save*.
. Scroll down to the *Capability config* section.
```

File: authorization_services/topics/resource-server-enable-authorization.adoc [hunk 2]
```diff
@@ -1,7 +1,12 @@
 [[_resource_server_enable_authorization]]
 = Enabling authorization services
 
+ifeval::[{project_community}==true]
+To turn your OIDC Client Application into a resource server and enable fine-grained authorization, click the *Authorization Enabled* switch to *ON* then click *Save*.
```
Comment (andymunro) [inline authorization_services/topics/resource-server-enable-authorization.adoc]:
```suggestion
You can turn your OIDC client into a resource server and enable fine-grained authorization.

 .Procedure
  . Toggle *Authorization Enabled* to *ON*.
  . Click *Save*.
```

File: authorization_services/topics/resource-server-enable-authorization.adoc [hunk 3]
```diff
@@ -1,7 +1,12 @@
...
+endif::[]
+ifeval::[{project_product}==true]
 To turn your OIDC Client Application into a resource server and enable fine-grained authorization, select *Access type* *confidential* and click the *Authorization Enabled* switch to *ON* then click *Save*.
```
Comment (andymunro) [inline authorization_services/topics/resource-server-enable-authorization.adoc]:
```suggestion
You can now turn your OIDC client into a resource server and enable fine-grained authorization.
 .Procedure
  . Set *Access type* to *confidential*.
  . Toggle *Authorization Enabled* to *ON*.
  . Click *Save*.
```

File: authorization_services/topics/resource-server-create-client.adoc (hunk 1 above)
Comment (andymunro) [reply authorization_services/topics/resource-server-create-client.adoc:27]: Line 39 should be:
ifeval::[{project_product}==true]
. Click *Save*. 
+
The client is created and the client Settings page displays. 
endif::[]
ifeval::[{project_community}==true]
. Click *Save*. 
+
The client is created.
endif::[]

File: authorization_services/topics/resource-server-enable-authorization.adoc [hunk 4]
```diff
@@ -1,7 +1,12 @@
...
+endif::[]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-enable-authorization.adoc]:
```suggestion
+
```

File: authorization_services/topics/resource-server-import-config.adoc [hunk 5]
```diff
@@ -12,8 +12,12 @@ The configuration settings for a resource server (or client) can be exported and
 .Procedure
 
 . Navigate to the *Resource Server Settings* page.
```
Comment (andymunro) [inline authorization_services/topics/resource-server-import-config.adoc]:
```suggestion
. Click *Clients* in the menu.
. Click the client you created as a resource server.
```

File: authorization_services/topics/resource-server-import-config.adoc [hunk 6]
```diff
@@ -12,8 +12,12 @@ The configuration settings for a resource server (or client) can be exported and
...
+ifeval::[{project_community}==true]
+. Click the *Export* tab.
+endif::[]
+ifeval::[{project_product}==true]
 . Click the *Export Settings* tab.
```
Comment (andymunro) [inline authorization_services/topics/resource-server-import-config.adoc]:
```suggestion
. Click *Clients* in the menu.
. Click the client you created as a resource server.
```

File: authorization_services/topics/resource-server-import-config.adoc [hunk 7]
```diff
@@ -12,8 +12,12 @@ The configuration settings for a resource server (or client) can be exported and
...
-. On this page, click *Export*.
+endif::[]
 +
 .Export Settings
 image:{project_images}/resource-server/authz-export.png[alt="Export Settings"]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-import-config.adoc]:
```suggestion
image:{project_images}/resource-server/authz-export.png[Export Settings]
```

File: authorization_services/topics/resource-server-import-config.adoc [hunk 8]
```diff
@@ -31,4 +35,9 @@ You can import a configuration file for a resource server.
 .Import Settings
 image:{project_images}/resource-server/authz-settings.png[alt="Import Settings"]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-import-config.adoc]:
```suggestion
image:{project_images}/resource-server/authz-settings.png[Import Settings]
```

File: authorization_services/topics/resource-server-enable-authorization.adoc [hunk 9]
```diff
@@ -1,7 +1,12 @@
...
 
 .Enabling authorization services
 image:{project_images}/resource-server/client-enable-authz.png[alt="Enabling authorization services"]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-enable-authorization.adoc]:
```suggestion
image:{project_images}/resource-server/client-enable-authz.png[Enabling authorization services]
```

File: authorization_services/topics/resource-server-create-client.adoc [hunk 10]
```diff
@@ -10,12 +10,22 @@ The first step to enable {project_name} Authorization Services is to create the
 .Clients
 image:{project_images}/resource-server/client-list.png[alt="Clients"]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-create-client.adoc]:
```suggestion
image:{project_images}/resource-server/client-list.png[Clients]
```

File: authorization_services/topics/resource-server-create-client.adoc [hunk 11]
```diff
@@ -10,12 +10,22 @@ The first step to enable {project_name} Authorization Services is to create the
...
 
+ifeval::[{project_community}==true]
+. On this page, click *Create client*.
+endif::[]
+ifeval::[{project_product}==true]
 . On this page, click *Create*.
+endif::[]
 +
 .Add Client
 image:{project_images}/resource-server/client-create.png[alt="Add Client"]
```
Comment (andymunro) [inline authorization_services/topics/resource-server-create-client.adoc]:
```suggestion
image:{project_images}/resource-server/client-create.png[Add Client]
```

**Review by andymunro — CHANGES_REQUESTED** [review]
Looks good, @edewit. Just a few changes.

**Review by andymunro — APPROVED** [review]

**Comment by andymunro** [comment]
@edewit looks good. Ready to merge, @ssilvert

**Review by ssilvert — APPROVED** [review]

**Pull request merged**